bin/sidecli.re

```diff
@@ -806,6 +807,104 @@ let self = {
   Term.(lwt_ret(const(self) $ folder_dest));
 };
 
+let add_trusted_validator = (folder, address) => {
+  open Networking.Trusted_validators_membership_change;
+  let file = folder ++ "/identity.json";
+  let.await identity = Files.Identity.read(~file);
+  let payload = Trusted_validators_membership_change.{address, action: Add};
+  let payload_json_str =
+    payload
+    |> Trusted_validators_membership_change.to_yojson
+    |> Yojson.Safe.to_string;
+  switch (payload_json_str |> BLAKE2B.of_string) {
+  | None =>
+    await(`Error((false, "Internal Error: Failed to compute BLAKE2B")))
```

Comment:
This doesn't make sense you're not hashing it.

Reply:
Making it "Failed to obtain Blake2b from string". Feel free to offer alternatives otherwise.

Reply:
You're not computing `BLAKE2B`, I believe you're looking for the `BLAKE2B.hash` function.

Reply:
Indeed. Fixed in the next commit